comictalker/resulttypes.py

```diff
@@ -33,7 +37,14 @@ class ComicIssue:
     id: str
     image_url: str
     issue_number: str
+    critical_rating: float
+    maturity_rating: str
+    manga: str
+    genres: list[str]
+    tags: list[str]
     name: str
+    language: str  # 2-letter iso code
+    country: str  # 2-letter iso code
```

Comment:
Same as before only the type will be enforced here. This should only exist between the cache and a single talker

Reply:
I suppose really this should be validated by the outputting module (CBI, ComicInfo, etc.).